code/game/machinery/kitchen/microwave.dm

```diff
@@ -308,9 +311,20 @@
 		stop()
 		if(cooked)
 			cooked.loc = src.loc
+			convert_nutriment(cooked)
 		return
 
-/obj/machinery/microwave/proc/wzhzhzh(var/seconds as num) // Whoever named this proc is fucking literally Satan. ~ Z
+/obj/machinery/microwave/proc/convert_nutriment(atom/target)
+	if(blacklisted_from_buffs.Find(target.type))
```

Comment:
You can use `in` instead of `find` if you don't care about the items index in the list.